src/track/beats.h

```diff
@@ -31,6 +31,11 @@ namespace mixxx {
 /// plain copyable, movable object.
 class BeatsInternal {
   public:
+    // TODO(hacksdump): These versions are retained for backward compatibility.
+    // In future, There will be no versions at all.
```

Comment:
We should still have versions. Better to add them now and not need them later than wish we had added them in the future.